DataModel/Protocol/Protocol-definition.yml

```diff
@@ -0,0 +1,285 @@
+Protocol:
+  description: Describes the protocols followed in a study, including details such
+    as the cell capture system, library protocol, the chemistry kit and sampling technique
+    used, as well as sequencing related information.
+  properties:
+    donor_id:
+      description: Foreign key to map to donor ID
+      type: string
+      format: alphanumeric
+      pattern: '"^DO-[0-9]{4}$"'
+    project_id:
+      description: Foreign key to map a project
+      type: string
+      format: alphanumeric
+      pattern: '"^P[0-9]{3}$"'
+    protocol_id:
+      description: Primary key for the protocol ID
+      type: string
+      format: alphanumeric
+      pattern: '"^PRT[0-9]{3}$"'
+    protocol_name:
+      description: Name given to the protocol such as single cell sequencing protocol,
+        Cytokine staining Protocol
+      type: string
+      format: text
+    protocol_description:
+      description: Description that provides enough information about the protocol
+      type: string
+      format: text
+    protocol_core:
+      description: Core protocol-level information
+      type: string
+      format: text
+    computational_method:
+      description: A URL to a versioned workflow and versioned execution environment
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/MMO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMMO_0000575
+    matrix_transformation:
+      description: Information related to protocols that transforms a matrix
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/ATOM?p=classes&conceptid=http%3A%2F%2Furi.interlex.org%2Ftgbugs%2Furis%2Freadable%2Fatlas%2FTransformationMatrix
+    bioinformatics_software:
+      description: Name of software used
```

Comment:
@ishitasati11 This property stores the description of the attribute and not the value it stores. So the description could be something like
`A wide array of computer programs and tools designed to manage, analyze, and interpret biological data. `